rt_entt_codegen/core/print_sys_exec.cc

```diff
@@ -171,35 +172,43 @@ static auto print_sys_exec_ctx_remove(
 			type_name,
 			">(this, ecsact_id_cast<ecsact_component_like_id>(",
 			type_name,
-			"::id));\n"
+			"::id), *view);\n"
```

Comment:
shouldn't this be `view_name`?
```suggestion
			"::id), *", view_name, ");\n"
```